eng/pipelines/common/platform-matrix.yml

```diff
@@ -223,21 +223,25 @@ jobs:
         ${{ insert }}: ${{ parameters.jobParameters }}
 
 # Windows arm
-
-- ${{ if or(containsValue(parameters.platforms, 'Windows_NT_arm'), eq(parameters.platformGroup, 'all')) }}:
-  - template: xplat-setup.yml
-    parameters:
-      jobTemplate: ${{ parameters.jobTemplate }}
-      helixQueuesTemplate: ${{ parameters.helixQueuesTemplate }}
-      osGroup: Windows_NT
-      archType: arm
-      platform: Windows_NT_arm
-      jobParameters:
-        stagedBuild: ${{ parameters.stagedBuild }}
-        buildConfig: ${{ parameters.buildConfig }}
-        helixQueueGroup: ${{ parameters.helixQueueGroup }}
-        managedTestBuildOsGroup: Windows_NT
-        ${{ insert }}: ${{ parameters.jobParameters }}
+# Currently down as there's no suitable ARM32 pool at the moment, tracked under
```

Comment:
Is this the best way to disable it? 

We still want to build the installer for `Windows_NT_arm`. What I'm currently doing in my single pipeline PR is to just exclude the test runs but still do builds (which make sense to do).

Reply:
Hmm, you're probably right, I'll modify the change as you suggest.